zhaquirks/visonic/mct340e.py

```diff
@@ -31,40 +31,16 @@
 DIAGNOSTICS_CLUSTER_ID = 0x0B05  # decimal = 2821
 
 
-class MCT340E(CustomDevice):
```

Comment:
I'd like a hard look at this in particular as i think i replaced this correctly to utilize the default function, but i also dont want to break anything. :-)